OrbitGl/AsyncTrack.cpp

```diff
@@ -16,8 +16,8 @@ using orbit_client_protos::TimerInfo;
 
 AsyncTrack::AsyncTrack(TimeGraph* time_graph, const std::string& name, OrbitApp* app)
     : TimerTrack(time_graph, app) {
-  SetName(name);
-  SetLabel(name);
+  name_ = name;
+  label_ = name;
```

Comment:
Personal preference: I would prefer to initialize these in the member initializer list, something like:
`: TimeTrack(time_graph, app), name_{name}, label_{name} { }`

Reply:
But that's just personal preference and since it has been like so in the original code, I'm also fine if you just leave it.